src/basho_bench_terminator.erl

```diff
@@ -1,4 +1,10 @@
--module(basho_bench_duration).
+%% -------------------------------------------------------------------
+%% Check whether to exit besho_bench, exit conditions
+%%     1. run out of duration time
+%%     2. run out of operations even if the duration time had not up
+%% -------------------------------------------------------------------
+
+-module(basho_bench_terminator).
```

Comment:
This commit should be two commits: 1) doing the rename from basho_bench_duration to basho_bench_terminator; 2) adding the additional logic. We do not mix whitespace/formatting/renaming/refactoring commits with actual logic changes, and in general it would be good to separate out the two into different PRs.